README.adoc

```diff
@@ -1,3 +1,121 @@
-Spring Cloud Data Flow Admin for Cloud Foundry
-----------------------------------------------
+= Spring Cloud Data Flow Admin for Cloud Foundry
+
+Spring Cloud Data Flow can be used to deploy modules in a Cloud Foundry environment. When doing so, the
+https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-admin[Admin]
```

Comment:
Ah, good point. I got carried away by asciidoc conversion, did not actually read what I was copy/pasting. Will update